webapp/public/js/domjudge.js

```diff
@@ -599,6 +600,20 @@ function updateMenuInternalErrors(data)
     }
 }
 
+function updateMenuBalloons(data)
```

Comment:
It sends a notification but does not update the menu. Do we want to add a badge to the menu with the number of undelivered balloons? Like clarifications?

Reply:
I was indeed planning that in a seperate commit as it was not discussed to do this yet. But my intentions yesterday were to indeed introduce such a method